plugins/wallabag.koplugin/main.lua

```diff
@@ -780,6 +801,39 @@ function Wallabag:addArticle(article_url)
     return self:callAPI("POST", "/api/entries.json", headers, body_JSON, "")
 end
 
+function Wallabag:addTags(path)
+    logger.dbg("Wallabag: managing tags for article ", path)
+    local id = self:getArticleID(path)
+    if id then
+        local docinfo = DocSettings:open(path)
+
+        local tags = docinfo.data.summary.note
+
+        if tags ~= "" and tags ~= nil then
+
+            logger.dbg("Wallabag: sending tags ", tags, " for ", path)
+
+            local body = {
+                tags = tags
+            }
+
+            local bodyJSON = JSON.encode(body)
+
+            local headers = {
+                ["Content-type"] = "application/json",
+                ["Accept"] = "application/json, */*",
+                ["Content-Length"] = tostring(#bodyJSON),
+                ["Authorization"] = "Bearer " .. self.access_token,
+            }
+
+            self:callAPI("POST", "/api/entries/" .. id .. "/tags.json", headers, bodyJSON, "")
+        else
+          logger.dbg("Wallabag: no tags to send for ", path)
```

Comment:
Odd, it still says unauthorized. Oh well, no matter. Btw, one tiny nit but I'll just add it through the GH web UI. :-)
```suggestion
            logger.dbg("Wallabag: no tags to send for ", path)
```